lib/src/map/options/map_options.dart

```diff
@@ -106,6 +83,8 @@ class MapOptions {
 
   final InteractionOptions interactionOptions;
 
+  /// The options of the closest [FlutterMap] ancestor. If this is called from a
```

Comment:
Is this unfinished?

Reply:
Missed that line while refactoring, good spot.